common/skiplinks/index.json-ld

```diff
@@ -0,0 +1,47 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Skip links",
+		"fr": "Liens de saut de page"
+	},
+	"description": {
+		"en": "Documentation and working example on how to use the skiplinks.",
+		"fr": "Documentation et example pratique sur l'utilisation des liens de saut de page."
+	},
+	"modified": "2024-02-27",
+	"componentName": "skiplinks",
+	"processing": "baseline",
+	"status": "stable",
+	"pages": {
+		"examples": [
+			{
+				"title": "Skip links",
+				"language": "en",
+				"path": "skiplinks-en.html"
+			},
+			{
+				"title": "Liens de saut de page",
+				"language": "fr",
+				"path": "skiplinks-fr.html"
+			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility assessment #1 - Skip links",
+				"language": "en",
+				"path": "reports/a11y-1-en.html"
+			},
+			{
+				"title": "Évaluation d'accessibilité #1 - Liens de saut de page",
+				"language": "fr",
+				"path": "reports/a11y-1-fr.html"
```

Comment:
```suggestion
				"path": "reports/ally-1-fr.html"
```

Reply:
switched all to use a11y for consistency